package.json

```diff
@@ -50,7 +50,14 @@
     "elliptic@6.5.4": "^6.5.7",
     "fast-xml-parser@^4.3.4": "^4.4.1",
     "send": "^0.19.0",
-    "ws@7.4.6": "^7.5.10"
+    "ws@7.4.6": "^7.5.10",
+    "minimist": "1.2.6",
+    "node-fetch": "^2.6.7",
+    "xmldom": "npm:@xmldom/xmldom@0.7.13",
+    "@walletconnect/socket-transport/ws": "^7.5.10",
+    "@ethersproject/providers/ws": "^7.5.10",
+    "puppeteer-core/ws": "^8.17.1",
+    "react-native/ws": "^6.2.3"
```

Comment:
It doesn't look like `react-native/ws` is in our dependency tree 

![Screenshot 2025-03-06 at 3 24 52 PM](https://github.com/user-attachments/assets/199a6585-840b-495b-a695-cd08deebae0c)